app/mailers/patient_mailer.rb

```diff
@@ -195,6 +208,18 @@ def closed_email(patient)
 
   private
 
+  # Mark the last_assessment_reminder_sent timestamp for a given patient to now. This will make the patient inelgibile
+  # to receive another message in the same day.
+  def update_last_assessment_reminder_sent(patient)
+    patient.update(last_assessment_reminder_sent: DateTime.now)
+  end
+
+  # Clear out the last_assessment_reminder_sent timestamp for a given patient so they will still be eligible for another
+  # report attempt today.
+  def clear_last_assessment_reminder_sent(patient)
+    patient.update(last_assessment_reminder_sent: nil)
+  end
```

Comment:
Do we need methods to update single attributes? If so, I feel like these methods would be suited for the Patient model.

Reply:
Originally my intention was for these methods to do more, but since that isn't the case I agree they probably aren't necessary. I'd rather not have these sorts of helpers on the patient model - since it's simple enough to do an update for a single field like this. Made this change.